integrations/logging/splunk/install.sh

```diff
@@ -12,6 +12,9 @@ cd ${SCRIPT_DIR}
 
 NAMESPACE="splunk-operator"
 
+set -o errexit
+set -o xtrace
```

Comment:
I you are adding this here, I guess that you'd want to add the other changes of this file in https://github.com/kubearchive/kubearchive/pull/776/files#diff-993791378b7246b0c4d06335d979f7e5b3b9004bba12f6bda04b224bb4cc2cc6, right?:

```
kubectl wait -n ${NAMESPACE} pod --all --for=condition=ready --timeout=90s
```

Reply:
I'm not sure, I didn't want to introduce a bunch of changes... let me try this and lets see.

Reply:
I increased it to 180s as it failed a couple of times with 90s.